### PR TITLE
docs(rest): add jina logo to redoc

### PR DESCRIPTION
**Pull request by deepankarm**
image

### CODE REVIEW

**Comment by hanxiao** [comment]
i think it's too big let me provide you an image with larger margin

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Review by hanxiao — CHANGES_REQUESTED** [review]
pic too big

**Comment by deepankarm** [comment]
> pic too big

I'm putting this in draft to avoid anyone merging it.

File: daemon/__init__.py [hunk 1]
```diff
@@ -70,6 +70,9 @@ def _get_app():
 def _write_openapi_schema(filename='daemon.json'):
     app = _get_app()
     schema = app.openapi()
+    schema['info']['x-logo'] = {
+        'url': 'https://api.jina.ai/logo/logo-company/logo/light-bg/Company%20logo_light@1x.png'
```
Comment (hanxiao) [inline daemon/__init__.py:74]:
```suggestion
        'url': 'https://api.jina.ai/logo/logo-product/jina-core/horizontal-layout/colored/Product%20logo_Core_vertical_colorful%402x-margin.png'
```

File: scripts/get-gateway-openapi.py [hunk 2]
```diff
@@ -9,5 +9,8 @@
 
 app = get_fastapi_app(args, logger)
 schema = app.openapi()
+schema['info']['x-logo'] = {
+    'url': 'https://api.jina.ai/logo/logo-company/logo/light-bg/Company%20logo_light@1x.png'
```
Comment (hanxiao) [inline scripts/get-gateway-openapi.py:13]:
```suggestion
    'url': 'https://api.jina.ai/logo/logo-product/jina-core/horizontal-layout/colored/Product%20logo_Core_vertical_colorful%402x-margin.png'
```

**Pull request merged**